apps/idam/ithc/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
```

Comment:
with `- ../../base` it will attempt all entries in this file - https://github.com/hmcts/cnp-flux-config/blob/master/apps/idam/base/kustomization.yaml

If only migrating/testing` idam-web-admin` you will need to add in this file, only what is required - 
https://github.com/hmcts/cnp-flux-config/blob/master/apps/idam/base/kustomization.yaml#L4
https://github.com/hmcts/cnp-flux-config/blob/master/apps/idam/base/kustomization.yaml#L6
https://github.com/hmcts/cnp-flux-config/blob/master/apps/idam/base/kustomization.yaml#L9 
^ Please review folder structure, more` ../ `will be required

This will temporary until other apps are migrated

Reply:
The problem with that is we already use kustomize on preview/sbox environments. I can't edit this file without affecting those other deployments.

Reply:
As discussed, to use migrate-base as an interim to review how Fluxv2 interacts. Not long term, long term will be `../../base`

Reply:
Done